### PR TITLE
fix: e2e flakiness

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by lukicenturi** [comment]
https://stackoverflow.com/a/58302240/8371889
This solution seems to work for most people with same issue, to make sure that the button is clicked

**Comment by kelsos** [comment]
> https://stackoverflow.com/a/58302240/8371889 This solution seems to work for most people with same issue, to make sure that the button is clicked

As long as we don't force anything 😄.

File: frontend/app/tests/e2e/pages/rotki-app.ts [hunk 1]
```diff
@@ -128,4 +128,23 @@ export class RotkiApp {
       expect(query.getAll(key)).to.deep.equal(values);
     });
   }
+
+  static navigateMenu(menu: string, submenu?: string) {
```
Comment (kelsos) [inline frontend/app/tests/e2e/pages/rotki-app.ts:132]: nit: can you rename it to `navigateTo`

**Review by kelsos — APPROVED** [review]
Thanks @lukicenturi lgtm, code looks good and PR is green

**Pull request merged**